src/main/java/org/openlmis/requisition/dto/RequisitionsProcessingStatusDto.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+package org.openlmis.requisition.dto;
+
+import org.apache.commons.lang3.BooleanUtils;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ * Represents the results of a batch processing of requisitions.
+ */
+@Getter
+public class RequisitionsProcessingStatusDto {
+
+  private List<ApproveRequisitionDto> requisitionDtos;
+  private List<RequisitionErrorMessage> requisitionErrors;
+
+  public RequisitionsProcessingStatusDto() {
+    this.requisitionDtos = new ArrayList<>();
+    this.requisitionErrors = new ArrayList<>();
+  }
+
+  public void addProcessedRequisition(ApproveRequisitionDto requisitionDto) {
+    requisitionDtos.add(requisitionDto);
+  }
+
+  public void addProcessingError(RequisitionErrorMessage errorMessage) {
+    this.requisitionErrors.add(errorMessage);
+  }
+
+  /**
+   * Removes skipped products but only if those products are skipped in all requisitions.
```

Comment:
From a user point of view those two thinks are the same because Requisition Line Item is related with single product so when we remove product we have to remove line item  (same opposite).